puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/foreman-plugins-pull-request.yaml

```diff
@@ -0,0 +1,73 @@
+- builder:
+    name: plugin-pull-request
+    builders:
+      - shell: |+
+          #!/bin/bash -ex
+
+          # check out the pull request
+          [ -e plugin ] && rm -rf plugin/
```

Comment:
`rm -rf plugin/` will also exit with 0 if it doesn't exist.

Reply:
oh nice! that's much simpler

Reply:
updated